src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export_features.json

```diff
@@ -1,11 +1,11 @@
 {
   "Color": "Red",
   "Region": "West US",
-  "feature_management": {
+  "FeatureManagement": {
```

Comment:
Does this mean `2020-08-01` will cause breaking change in output format of `appconfig kv export`?

Reply:
When I run live test for appconfig, the .json file is changed, so I add it.